tests/agent/message_service/test_subscriber.py

```diff
@@ -0,0 +1,82 @@
+from unittest.mock import ANY, Mock
+
+import pytest
+
+from great_expectations.agent.message_service.rabbit_mq_client import RabbitMQClient
+from great_expectations.agent.message_service.subscriber import (
+    EventContext,
+    Subscriber,
+)
+from tests.agent.message_service.amqp_errors import get_amqp_errors
+
+
+def test_subscriber_has_client_attribute():
+    client = Mock(autospec=RabbitMQClient)
+
+    subscriber = Subscriber(client=client)
+
+    assert subscriber.client is client
```

Comment:
This seems like an unnecessary test.
The type-checker verifies the attribute is present.